lms/djangoapps/support/tests/test_views.py

```diff
@@ -44,6 +44,48 @@ def setUp(self):
         self.assertTrue(success, msg="Could not log in")
 
 
+class SupportViewManageUserTests(SupportViewTestCase):
+    """
+    Base class for support view tests.
+    """
+
+    def setUp(self):
+        """Make the user support staff"""
+        super(SupportViewManageUserTests, self).setUp()
+        SupportStaffRole().add_users(self.user)
+
+    def test_get_support_form(self):
+        """
+        Tests Support View to return Manage User Form
+        """
+        url = reverse('support:manage_user')
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
```

Comment:
add test to make sure if user role is not SupportStaff response.status_code is not 200

Reply:
i have added the newly created urls in the following class which is already doing your requested change ..
https://github.com/edx/edx-platform/blob/master/lms/djangoapps/support/tests/test_views.py#L49